docs/source/gateway/index.rst

```diff
@@ -410,6 +414,10 @@ below can be used as a helpful guide when configuring a given route for any newl
      - OpenAI
      - gpt-3.5-turbo, gpt-4
      - Yes
+   * - llm/v1/chat
```

Comment:
Can you please also add a blurb (rst Note) somewhere:
"Customers are responsible for ensuring compliance with applicable model licenses."

Probably up top before we list all the modes.